settings.gradle.kts

```diff
@@ -146,13 +146,16 @@ fun loadProjects(file: String) {
 
 loadProjects("gradle/projects.main.properties")
 
-val ideaSyncActive = System.getProperty("idea.sync.active").toBoolean()
+val ideSyncActive =
+  System.getProperty("idea.sync.active").toBoolean() ||
+    System.getProperty("eclipse.product") != null ||
+    gradle.startParameter.taskNames.any { it.startsWith("eclipse") }
```

Comment:
just wondering:
could the IDEA detection be part of https://github.com/projectnessie/gradle-build-plugins somehow?

Reply:
Nope - well, maybe.
Would need to build two plugins (one for a build.gradle + one for a settings.gradle).